docs/Plugins/pagerduty.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Pagerduty (WIP)"
+description: >
+  Pagerduty Plugin
+---
+
+
+
+## Summary
+
+This plugin collects all incidents from Pagerduty, and uses them to compute incident-type DORA metrics.
```

Comment:
How about:
"..., and uses them to compute DORA metrics such as [Median Time to Restore Service](https://devlake.apache.org/docs/Metrics/MTTR) and [Change Failure Rate](https://devlake.apache.org/docs/Metrics/CFR)."